src/lib/translations/en.json

```diff
@@ -13,6 +13,10 @@
 		"vatNumber": "VAT number",
 		"zipCode": "Zip code"
 	},
+	"ageWarning": {
+		"agreement": "Compris chef",
+		"navigate": "Continuer la navigation"
```

Comment:
Those are english wordings in French. Check the other translations too, like italian.

Btw @Tirodem do we really want to use the "compris chef" in  the wording?

Reply:
so a sub-param in `runtimeConfig.ageRestriction` @ithiame

Reply:
Ah, there's no translation for /config so it would be bothersome, so here's the translation key... ok.
Let me think of a wording.

Reply:
"J'ai pris connaissance des conditions d'accès, les remplit et les respecte."
"I have read the access conditions, fulfill them and respect them."
+ other languages